plugins/modules/junos_bgp_global.py

```diff
@@ -1404,36 +1433,42 @@
 #         "preference": "2"
 #     }
 #
-
 #
 # Using purged
 #
 # Before state
 # ------------
 #
 # admin# show protocols bgp
-# description "This is configured with Junos_bgp resource module replace";
+# precision-timers;
+# advertise-from-main-vpn-tables;
+# holddown-all-stale-labels;
+# description "This is configured with Junos_bgp resource module";
+# accept-remote-nexthop;
 # preference 2;
 # hold-time 5;
 # advertise-inactive;
+# no-advertise-peer-as;
+# no-aggregator-id;
 # out-delay 10;
+# log-updown;
+# damping;
 # bgp-error-tolerance {
-#     malformed-route-limit 40000000;
+#     malformed-route-limit 20000000;
 # }
 # authentication-algorithm md5;
-# advertise-bgp-static {
-#     policy static-to-bgp;
-# }
-# bfd-liveness-detection {
-#     version automatic;
-#     minimum-receive-interval 8;
-#     multiplier 30;
-#     no-adaptation;
-#     transmit-interval {
-#         minimum-interval 4;
-#     }
+# no-client-reflect;
+# include-mp-next-hop;
+# bmp {
+#     monitor enable;
 # }
+# add-path-display-ipv4-address;
```

Comment:
An example that shows neighbor configuration and as_number would be great.

Reply:
yeah, that would be good to have that updated for merged and neighbor for delete operation example.